src/main/java/nextstep/subway/domain/Section.java

```diff
@@ -35,4 +34,21 @@ public Section(Line line, Station upStation, Station downStation, int distance)
         this.downStation = downStation;
         this.distance = distance;
     }
+
+    public boolean isSameSection(Section newSection) {
+        return this.getUpStation() == newSection.getUpStation()
+                && this.getDownStation() == newSection.getDownStation();
+    }
+
+    public boolean isSameDownStation(Station downStation) {
+        return this.getDownStation() == downStation;
+    }
+
+    public boolean isSameUpStation(Station upStation) {
+        return this.getUpStation() == upStation;
+    }
+
+    public boolean isLongerThan(Section section) {
+        return section.getDistance() - this.getDistance() <= 0;
```

Comment:
Distance를 하나의 클래스로 분리한다면 거리를 비교하는 책임을 Distacne에게 위임할 수 있을 것 같은데 어떻게 생각하시나요?
거리를 비교하는 로직도 연산 이후 비교하는 것보다 직접 논리 연산을 이용하여 비교하는 것이 직관적인 것 같은데 이 부분도 어떻게 생각하시는지 궁금합니다 🤔 
```suggestion
        return this.getDistance() > section.getDistacne();
```

Reply:
연산 이후에 하니 확실히 가독성이 떨어진게 고민이긴했어요. 
0을 이하인지 기준으로 생각할게 아니라 대소로 하면 확실히 직관적 이네요. 좋은생각입니다.